common/djangoapps/django_sudo_helpers/settings.py

```diff
@@ -0,0 +1,8 @@
+"""
+Settings for the django-sudo module.
+"""
+
+
+def apply_settings(django_settings):
+    """Set provider-independent settings."""
+    django_settings.SOCIAL_AUTH_PIPELINE += ('django_sudo_helpers.pipeline.create_sudo_session',)
```

Comment:
The exact ordered list of what steps are in the pipeline is quite important to security of third_party_auth, and I don't really like having the pipeline setting being sometimes modified by code elsewhere in the codebase. I'd prefer that you put `if using_django_sudo: django_settings.SOCIAL_AUTH_PIPELINE += ('django_sudo_helpers.pipeline.create_sudo_session',)` right into `third_party_auth/settings.py` where the rest of the pipeline steps are all configured, or at least add a comment there explaining that sometimes an extra pipeline step will be added.